bsip-0062.md

```diff
@@ -0,0 +1,43 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
+  Authors:
+    Stefan Schießl stefan.schiessl@blockchainprojectsbv.com
+    Jerry Liu bitcrab@qq.com
+  Status: Draft
+  Type: Protocol
+  Created: 2019-09-14
+  Worker: TBD
+  ```
+
+# Abstract
+Closing a short position comes with a risk of getting margin called if no spare funds are available to buy the long and reduce the debt to zero. This BSIP introduces a way to put the short position directly on the market.
+
+# Rationale
+Closing a short position with a healthy CR should come with no risk of getting margin called.
+
+# Specification
+Introduce a way that user can directly place orders to sell collateral in a margin position, and the received SmartCoins will be used to reduce the debt automatically.
```

Comment:
A possible implementation would be an extension to `limit_order_create`that points to the debt position. This would be similar to the `limit_order` extension specified in BSIP-70 (trading from a loan portfolio).

Reply:
Would this approach also allow a dynamic price through specifying the offset from feed_price and the target to receive amount?

Reply:
Probably not.

Reply:
I suppose it's fine to have both ideas .. I added it to the BSIP